brian2/spatialneuron/morphology.py

```diff
@@ -782,7 +790,8 @@ def start_z(self):
         '''
         return _to_meters(self.start_z_)
 
-    @abc.abstractproperty
+    @property
+    @abc.abstractmethod
```

Comment:
What's the effect of this change? (Just so I can understand.)

Reply:
There's no real effect. Before Python 3.3, you could not use `@abc.abstractmethod` with a property and you'd have to use `abc.abstractproperty` instead. Since Python 3.3, you can combine `abstractmethod` with `property`, and `abstractproperty` has therefore been deprecated. It is still present in Python 3.9, though.

If I understand correctly, there's actually one advantage over `abstractproperty`: you could make only either the property setter or getter abstract. But we are not using this.